priv/repo/migrations/20220908132321_add_voice_text_chat.exs

```diff
@@ -1,4 +1,4 @@
-defmodule Ret.Repo.Migrations.AddFlyPermission do
+defmodule Ret.Repo.Migrations.AddVoiceTextChatPermission do
```

Comment:
based on the filename, this seems like a correct (unrelated) change. Just making sure the module name doesn't impact existing applied migrations or rollbacks at all.

Reply:
I’m glad you brought it up. The module name doesn’t impact the existing applied migrations. The reason is that those are keyed with the unique version (e.g., `20220908132321`), which happens to be a timestamp, in the `schema_migrations` table managed by Ecto.